WooCommerce/WooCommerceTests/ViewRelated/Feature Highlight/ProductDescriptionAITooltipUseCaseTests.swift

```diff
@@ -0,0 +1,84 @@
+import XCTest
+@testable import WooCommerce
+
+final class ProductDescriptionAITooltipUseCaseTests: XCTestCase {
+    // MARK: isDescriptionAIEnabled
+    func test_shouldShowTooltip_is_true_only_when_isDescriptionAIEnabled_is_true() {
```

Comment:
Nit: As commented on the previous PR, I think it doesn't make sense to create this use case when `isDescriptionAIEnabled` is false. We should consider removing it and move the check to outside of this class.

Reply:
Brought in latest changes from the previous PR and removed these tests in https://github.com/woocommerce/woocommerce-ios/pull/10026/commits/c305cab9059f89485dada7101afa1e63c10a1fba